shark_turbine/kernel/wave/utils.py

```diff
@@ -388,16 +388,11 @@ def compile_and_invoke(
                 )
         else:
             for inp in kernel_inputs:
-                tf = tempfile.NamedTemporaryFile()
-                torch.save(inp, tf)
+                tf = tempfile.NamedTemporaryFile(suffix=".npy")
+                numpy.save(tf, inp.numpy())
```

Comment:
is this required because iree-benchmark-module expects np files?

Reply:
Not sure what was broken exactly, benchmarking module wasn't throwing any errors, it was just returning empty benchmarking info.